code/src/components/Game.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import StartScreen from './StartScreen';
+import GameScreen from './GameScreen';
+import { Loader } from './Loader';
+
+const Game = () => {
+    const username = useSelector(store => store.game.username);
+    const isLoading = useSelector(store => store.game.isLoading);
+
+    return (
+        <div>
+            {isLoading && <Loader />}
+            {!isLoading && (
```

Comment:
I love the mario loader it looks really good in the short glimpses I get!! ;D
Would maybe be nice to add a little message that pops up if you don't write a username, like a 'please fill in your name' or something for when you try to start the game without entering a name.